parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -159,13 +159,6 @@ write_files:
   content: !!binary |
     {{CloudInitData "labelNodesSystemdService"}}
 
-- path: /etc/systemd/system/kms.service
```

Comment:
So basically we are deprecating all systemd foo in favor of everything being deliverd via daemonset?

Reply:
Actually nevermind I see it's a system Pod